src/mscorlib/shared/System/Diagnostics/Debug.cs

```diff
@@ -95,7 +95,7 @@ public static void Assert(bool condition, string message, string detailMessage)
                 string stackTrace;
                 try
                 {
-                    stackTrace = new StackTrace(0, true).ToString(System.Diagnostics.StackTrace.TraceFormat.Normal);
+                    stackTrace = new StackTrace(0, true).ToString();
```

Comment:
Default ToString() seems to have an extra new line at the end. Have you checked that the final output does not look odd with it?

Reply:
Here is what the output looks like:
```
E:\test\issue14867\testConsole\bin\Debug\netcoreapp2.0\win7-x64\publish>corerun testConsole.dll
Assertion Failed


   at debug3.Program.Main(String[] args) in E:\test\issue14867\debug3\Program.cs:line 11


E:\test\issue14867\testConsole\bin\Debug\netcoreapp2.0\win7-x64\publish>
```

I guess it does look sort of strange with the double newlines between them. This might need some change in the native part of the code. I'll follow up another PR with that change..